### PR TITLE
10683 - Activity log fixes, based off feedback

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/10683

*Description of changes:*
Changes based on Feedback.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/src/auth_api/services/org.py [hunk 1]
```diff
@@ -233,11 +234,12 @@ def _create_payment_settings(org_model: OrgModel, payment_info: dict,  # pylint:
         else:
             payment_account_status = PaymentAccountStatus.FAILED
 
-        if payment_account_status != PaymentAccountStatus.FAILED:
+        if payment_account_status != PaymentAccountStatus.FAILED and payment_method:
+            payment_method_description = PaymentMethod(payment_method).name if payment_method in [
```
Comment (seeker25) [inline auth-api/src/auth_api/services/org.py:238]: check to see if it's in enum before using it.

File: auth-web/src/services/org.services.ts [hunk 2]
```diff
@@ -82,7 +82,8 @@ export default class OrgService {
   }
 
   static async removeAffiliation (orgIdentifier: number, incorporationNumber: string, passcodeResetEmail?: string, resetPasscode?: boolean): Promise<AxiosResponse<void>> {
-    return axios.delete(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgIdentifier}/affiliations/${incorporationNumber}`, { data: { passcodeResetEmail: passcodeResetEmail, resetPasscode: resetPasscode } })
+    return axios.delete(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgIdentifier}/affiliations/${incorporationNumber}`,
+      { data: { passcodeResetEmail: passcodeResetEmail, resetPasscode: resetPasscode, logDeleteDraft: true } })
```
Comment (seeker25) [inline auth-web/src/services/org.services.ts:86]: `logDeleteDraft` to differentiate between GUI calls and backend calls (from other API's)

File: queue_services/business-events-listener/src/business_events_listener/worker.py [hunk 3]
```diff
@@ -134,7 +134,8 @@ async def process_name_events(event_message: Dict[str, any]):
                 affiliation: AffiliationModel = AffiliationModel(entity=nr_entity, org=org)
                 affiliation.flush()
                 activity: ActivityModel = ActivityModel(org_id=org.id, action=ActivityAction.CREATE_AFFILIATION.value,
-                                                        item_name=nr_entity.name, item_id=nr_entity.business_identifier,
```
Comment (seeker25) [inline queue_services/business-events-listener/src/business_events_listener/worker.py:137]: There is no name at this point only the identifier.. as the user can change the name until it's staff reviewed.

**Review by ritvick — APPROVED** [review]

**Comment by seeker25** [comment]
Holding off on merging until Namex on DEV is back up and running. There are some integration tests that need to be done.

**Pull request merged**